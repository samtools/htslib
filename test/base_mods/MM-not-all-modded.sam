@SQ	SN:I	LN:999
r1	0	I	1	0	36M	*	0	0	AGCTCTCCAGAGTCGNACGCCATYCGCGCGCCACCA	DF?GCH88.EG8.7@E9G8A?H9.:C?8,@,,9F@A	Mm:Z:C+m,2,2,1,4,1;C+h,6,7;N+n,15,2;	Ml:B:C,128,153,179,204,230,159,6,215,240
r1b	0	I	1	0	36M	*	0	0	AGCTCTCCAGAGTCGNACGCCATYCGCGCGCCACCA	DF?GCH88.EG8.7@E9G8A?H9.:C?8,@,,9F@A
r2	0	I	4	0	3S33M	*	0	0	AGCTCTCCAGAGTCGNACGCCATYCGCGCGCCACCA	DF?GCH88.EG8.7@E9G8A?H9.:C?8,@,,9F@A	Mm:Z:C+m,2,2,1,4,1;C+h,6,7;N+n,15,2;	Ml:B:C,128,153,179,204,230,159,6,215,240
r2b	0	I	4	0	3S33M	*	0	0	AGCTCTCCAGAGTCGNACGCCATYCGCGCGCCACCA	DF?GCH88.EG8.7@E9G8A?H9.:C?8,@,,9F@A
