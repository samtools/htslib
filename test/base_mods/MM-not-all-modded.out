0	A
1	G
2	C
3	T
4	C
5	T
6	C	C+m128
7	C
8	A
9	G
10	A
11	G
12	T
13	C
14	G
15	N	N+n215
16	A
17	C	C+m153
18	G	N+n240
19	C	C+h159
20	C	C+m179
21	A
22	T
23	Y
24	C
25	G
26	C
27	G
28	C
29	G
30	C
31	C	C+m204
32	A
33	C
34	C	C+m230 C+h6
35	A
---
Present: m. h. n.
6	C	C+m128
15	N	N+n215
17	C	C+m153
18	G	N+n240
19	C	C+h159
20	C	C+m179
31	C	C+m204
34	C	C+m230 C+h6

===

0	A
1	G
2	C
3	T
4	C
5	T
6	C
7	C
8	A
9	G
10	A
11	G
12	T
13	C
14	G
15	N
16	A
17	C
18	G
19	C
20	C
21	A
22	T
23	Y
24	C
25	G
26	C
27	G
28	C
29	G
30	C
31	C
32	A
33	C
34	C
35	A
---
Present:

===

0	A
1	G
2	C
3	T
4	C
5	T
6	C	C+m128
7	C
8	A
9	G
10	A
11	G
12	T
13	C
14	G
15	N	N+n215
16	A
17	C	C+m153
18	G	N+n240
19	C	C+h159
20	C	C+m179
21	A
22	T
23	Y
24	C
25	G
26	C
27	G
28	C
29	G
30	C
31	C	C+m204
32	A
33	C
34	C	C+m230 C+h6
35	A
---
Present: m. h. n.
6	C	C+m128
15	N	N+n215
17	C	C+m153
18	G	N+n240
19	C	C+h159
20	C	C+m179
31	C	C+m204
34	C	C+m230 C+h6

===

0	A
1	G
2	C
3	T
4	C
5	T
6	C
7	C
8	A
9	G
10	A
11	G
12	T
13	C
14	G
15	N
16	A
17	C
18	G
19	C
20	C
21	A
22	T
23	Y
24	C
25	G
26	C
27	G
28	C
29	G
30	C
31	C
32	A
33	C
34	C
35	A
---
Present:

===

